actions/subscribers.go

```diff
@@ -472,19 +472,9 @@ func ExportSubscribers(c *gin.Context) {
 		return
 	}
 
-	go func(c context.Context, report *entities.Report) {
-		err := reportSvc.GenerateExportReport(c, report)
-		if err != nil {
-			logger.From(c).WithFields(logrus.Fields{
-				"report": report,
-			}).WithError(err).Errorf("export failed")
-			//TODO discuss handling this error
-		}
-	}(c.Copy(), report)
+	go reportSvc.GenerateExportReport(c, report)
```

Comment:
@Gligor23 it was better to handle the error in the handler than in the service method, we should change it back. It's easier to test the service method that way if we want to.